### ISSUE TITLE
[Theme Request] Albedo from Overlord

### ISSUE THREAD

**Issue by farukaygun** [doki-theme/doki-master-theme#216]
## Who/What do you want to be featured?
[Albedo ](https://overlordmaruyama.fandom.com/wiki/Albedo) from Overlord

### What do they look like?
[image](https://static.wikia.nocookie.net/overlordmaruyama/images/c/c6/Albedo_Anime_Updated.png/revision/latest?cb=20220209052114)
[image](https://preview.redd.it/lftnkq48wc971.png?auto=webp&s=e929e08521224038483a611252ff1157b0539143)
[image](https://w7.pngwing.com/pngs/380/13/png-transparent-albedo-overlord-anime-chibi-overlord-vertebrate-fictional-character-bird-thumbnail.png)

## What type of theme?

- [] Light Theme
- [x] Dark Theme

## Additional Information (Optional)
If you can use monokai or dracula color tones as the background color, it is much more comfortable on the eyes.

Thanks

**Comment by Unthrottled** [comment]
> If you can use monokai or dracula color tones as the background color, it is much more comfortable on the eyes.

Sounds like you already know what you want :) 

Why not just use those color schemes & just put her as the background image in your editors?

**Comment by farukaygun** [comment]
> Why not just use those color schemes & just put her as the background image in your editors?

Yes, this is also an option but i said it only for background color.
